test/server.js

```diff
@@ -32,6 +32,10 @@ app.get('/', function (req, res) {
   res.sendFile(path.join(__dirname, 'public', 'app.html'))
 })
 
+app.get('/pixelmatch.js', function (req, res) {
+  res.sendFile(path.join(__dirname, '../node_modules/pixelmatch/', 'index.js'))
+})
+
```

Comment:
this is the simplest way i found to serve it from the node_modules folder. i think is ok.